CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 - PR #579 CMake: handle thrust via target
 - PR #581 Improve logging documentation
 - PR #585 Update ci/local/README.md
+- PR #586 Replaced `move` with `std::move`
```

Comment:
Once we fix this, CI should be able to start

```suggestion
- PR #587 Replaced `move` with `std::move`
```

Reply:
Ah yes, bad typos!

Reply:
No worries. This happens to me too as I try to guess what the number will be before submitting 😄